packages/mui-material/src/Slider/Slider.js

```diff
@@ -284,20 +245,7 @@ const SliderThumb = styled('span', {
   },
 }));
 
-SliderThumb.propTypes /* remove-proptypes */ = {
-  // ----------------------------- Warning --------------------------------
-  // | These PropTypes are generated from the TypeScript type definitions |
-  // |     To update them edit the d.ts file and run "yarn proptypes"     |
-  // ----------------------------------------------------------------------
-  /**
-   * @ignore
-   */
-  children: PropTypes.node,
-};
-
-export { SliderThumb };
-
-const StyledSliderValueLabel = styled(SliderValueLabel, {
+export const SliderValueLabel = styled(BaseSliderValueLabel, {
```

Comment:
Renaming it is a breaking change.

Reply:
Is it ?
The exported name was already `SliderValueLabel`

Reply:
Ah, ok, never mind. I thought this was the reason the build failed earlier.